src/main.js

```diff
@@ -1718,6 +1718,9 @@ const DiscoveryFeedApplication = new Lang.Class({
         // to it so that we can re-create it later
         this._window.connect('destroy', Lang.bind(this, function() {
             this._window = null;
+            this._discoveryFeedCardModel = new Gio.ListStore({
+                item_type: Stores.DiscoveryFeedCardStore.$gtype
+            });
```

Comment:
Instead of doing this, you could simply create the model together with the window, no?

Reply:
Yeah we could do that. My thought here was that the model was a property of the application and not the window, but the window is the only thing it is really used with...